react/NotFoundSearch.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types'
+import React, { Fragment } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { ExtensionPoint } from 'render'
+import { range } from 'ramda'
+
+const flexStyle = { flex: 1 }
+
+export default function NotFoundSearch({ term }) {
```

Comment:
Add documentation and you may extract the function and export it at the end of the file, so it would be easier to add hocs later